app/controllers/concerns/essi/works_controller_behavior.rb

```diff
@@ -23,16 +23,16 @@ def additional_response_formats(wants)
     # Overrides stock Hyrax method to accept retrieving a cached JSON manifest_builder
     def manifest
       headers['Access-Control-Allow-Origin'] = '*'
-  
-      json = sanitize_manifest(JSON.parse(manifest_builder))
-  
+      # manifest builder is now being handled in the IIIF Print gem
+      json = iiif_manifest_builder.manifest_for(presenter: presenter)
```

Comment:
If the manifest builder is changed to be handled by iiif_print, will it still cache manifests like or similar to the custom `manifest_builder` method?

Reply:
Also, what were we fixing by also running the resulting json through [custom sanitize methods](https://github.com/IU-Libraries-Joint-Development/essi/blob/main/app/controllers/concerns/essi/works_controller_behavior.rb#L58-L72)?  That has also been undone by the direct call to the iiif_print manifest builder.

Reply:
Info on how to implement Hyrax manifest caching feature has been captured in https://iu-uits.atlassian.net/browse/ESSI-1847 .  

But we should still think about any effects of losing the custom sanitize methods as noted above.